examples/nextjs/src/components/my-page.js

```diff
@@ -13,11 +12,24 @@ import Navigation from "./layout/navigation";
 import { usePathname, useRouter } from "next/navigation";
 import { DotcmsLayout } from "@dotcms/react";
 
+import { DotExperimentsProvider } from "@dotcms/experiments";
+
+
 export function MyPage({ data, nav }) {
-  const { refresh } = useRouter();
+  const { refresh, replace, route, asPath, query } = useRouter();
   const pathname = usePathname();
 
+  const experimentConfig = {
+    'api-key': process.env.NEXT_PUBLIC_EXPERIMENTS_API_KEY ,
+    server: process.env.NEXT_PUBLIC_DOTCMS_HOST ,
+    debug: process.env.NEXT_PUBLIC_EXPERIMENTS_DEBUG,
+    redirectFn: replace
+  }
+
   return (
+    // Provide the DotExperimentsProvider with the configuration and instance of DotExperiments
+    <DotExperimentsProvider config={experimentConfig} >
```

Comment:
Can this be moved to the sdk?
So that it is the same sdk that embeds everything within the DotExperimentsProvider
(if applicable)